debian/bionic/foreman/rules

```diff
@@ -16,22 +19,22 @@ build:
 	cd ./vendor/cache && /usr/bin/gem fetch thor -v '1.1.0'
 	/bin/rm -f bundler.d/test.rb bundler.d/development.rb
 	/bin/sed -i '2i  gem "puma-status"' bundler.d/service.rb
-	BUNDLE_BUILD__SASSC=--disable-march-tune-native BUNDLE_RETRY=5 BUNDLE_JOBS=4 /usr/bin/bundle install
-	/usr/bin/bundle package --no-install
+	BUNDLE_BUILD__SASSC=--disable-march-tune-native BUNDLE_RETRY=5 BUNDLE_JOBS=4 $(BUNDLE) install --path ./install-bundle/
```

Comment:
I do think `--path` is deprecated. It's now preferred to use `bundle config` or the `BUNDLE_PATH` env var. https://bundler.io/man/bundle-config.1.html#REMEMBERING-OPTIONS says it will be changed in Bundler 3.